src/components/statistics/DoomBar.js

```diff
@@ -1,69 +1,68 @@
 import { Jumbotron, Container, Row, Col, ProgressBar } from 'react-bootstrap'
-import { doomCounterValue } from './../helpers'
+import { doomCounterValue } from '../helpers'
 import { FaTwitterSquare } from 'react-icons/fa';
 import { updateWarning, newOrOld, percentageDifference } from '../helpers'
 
 function DoomBar() {
     const [steps, stepsProcessed, max, doomCounterValues, doomCounterlabels] = doomCounterValue()
     return (
         <Jumbotron className='p-0 m-2' >
-            <Container className="text-center pt-2" >
+            <Container className='text-center pt-2' >
                 <Row>
                     <Col>
                         <h4>Etapy zasad bezpieczeństwa</h4>
-                        <p className="font-weight-light">
+                        <p className='font-weight-light'>
                             Średnia liczba nowych zakażeń przez ostatnie 7 dni:
                             <b> {Math.round(newOrOld('the_average'))} {percentageDifference('the_average')}
                                 {updateWarning('the_average')}
                             </b>
                         </p>
-                        <ProgressBar className='doom-bar'>

```

Comment:
Tutaj w ogóle by można wykorzystać moduły CSS, np. zamiast "bar" mieć `classname={styles.bar}` gdzie styles to `import styles from './jakiśModuł.module.scss'`, wtedy się ładnie podpowiada :)

Reply:
Ofc jak na tak małą aplikację to imo nie ma nic złego w takim zwykłym nakładaniu tych klas jak tutaj, jest okej